lib/server-parser.js

```diff
@@ -0,0 +1,377 @@
+import fs from 'fs';
+import _ from 'lodash';
+import { DEFAULT_BASE_PATH } from 'appium-base-driver';
+
+// serverArgs will be added to the `server` (default) subcommand
+const serverArgs = [
```

Comment:
I think it makes sense to put server args into a separate json file

Reply:
they can't totally be in a json file since they reference js functions, but i moved them into their own file.